kubernetes/client/api_client.py

```diff
@@ -59,12 +59,14 @@ class ApiClient(object):
         'object': object,
     }
 
-    def __init__(self, configuration=None, header_name=None, header_value=None, cookie=None):
+    def __init__(self, configuration=None, header_name=None, header_value=None, cookie=None, async_req=True):
```

Comment:
This file is auto-generated by [swagger-codegen](https://github.com/kubernetes-client/gen). You could contribute to swagger-codegen [python template](https://github.com/swagger-api/swagger-codegen/blob/d5f36191998396f4810155dd19cecdf697a72e95/modules/swagger-codegen/src/main/resources/python/api_client.mustache). When you did, we can import the changes here. Ref: https://github.com/kubernetes-client/python/issues/226

Reply:
@roycaihw sure, can migrate the code change there, but other than that, the code looks ok?

Reply:
the code change makes sense to me, although I'm not a python expert :)

Reply:
@huang195 looks good to me. please do follow the path mentioned by @roycaihw

Reply:
I created the PR here: https://github.com/swagger-api/swagger-codegen/pull/7782